docs/package.json

```diff
@@ -82,8 +83,34 @@
   },
   "devDependencies": {
     "netlify-cli": "^2.59.0",
+    "remark-cli": "^8.0.1",
+    "remark-lint": "^7.0.1",
+    "remark-lint-no-dead-urls": "^1.1.0",
+    "remark-validate-links": "^10.0.2",
     "toml": "^3.0.0"
   },
+  "remarkConfig": {
+    "plugins": [
+      [
+        "remark-validate-links",
+        {
+          "prefix": "/docs/",
+          "lines": false
+        }
+      ],
+      "remark-lint",
+      [
+        "remark-lint-no-dead-urls",
```

Comment:
given the small number of external URLs today, I think it's fine to run it upon each merge. WDYT?

Reply:
lets do it for now and see if it becomes a problem? the only thing is adding rasa and rasa x features that link to each other 🤔

Reply:
yeah these might lock us a bit 😖